eng/cibuild_bootstrapped_msbuild.ps1

```diff
@@ -128,6 +128,9 @@ catch {
 }
 finally {
   Pop-Location
+
+  KillProcessesFromRepo
```

Comment:
It would be comical if this kills any lingering telemetry processes that are draining events, but afaik nothing like that exists.